packages/os/settings-applier.service

```diff
@@ -12,4 +12,4 @@ RemainAfterExit=true
 StandardError=journal+console
 
 [Install]
-WantedBy=multi-user.target
+WantedBy=preconfigured.target
```

Comment:
Previously, this unit was effectively `RequiredBy: configured.target`.
```suggestion
RequiredBy=preconfigured.target
```

Reply:
Done